### PR TITLE
feat / deconstructing compas-flow

### PR DESCRIPTION
**Pull request by samirromdhani**
fix RSR-1113

### CODE REVIEW

File: sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ied/InputsAdapterTest.java [hunk 1]
```diff
@@ -305,15 +326,4 @@ void filterDuplicatedExtRefs_should_not_remove_not_duplicated_extrefs() {
                 .hasSize(6);
     }
 
-    @Test
-    void updateAllExtRefIedNames_when_DOI_Mod_and_DAI_stVal_notExists_should_not_produce_error() {
-        // Given
-        SCL scl = SclTestMarshaller.getSCLFromFile("/scd-refresh-lnode/Test_Missing_ModstVal_In_LN0_when_binding.scd");
```
Comment (massifben) [inline sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ied/InputsAdapterTest.java:311]: Test file `/scd-refresh-lnode/Test_Missing_ModstVal_In_LN0_when_binding.scd` can be deleted

**Review by massifben — APPROVED** [review]

File: sct-commons/src/test/java/org/lfenergy/compas/sct/commons/ExtRefEditorServiceTest.java [hunk 2]
```diff
@@ -19,10 +18,8 @@
 import org.lfenergy.compas.sct.commons.model.epf.*;
 import org.lfenergy.compas.sct.commons.scl.ExtRefService;
```
[the remark this answers is not in the document]
Comment (samirromdhani) [reply sct-commons/src/test/java/org/lfenergy/compas/sct/commons/ExtRefEditorServiceTest.java:19]: done

File: sct-commons/src/main/java/org/lfenergy/compas/sct/commons/ExtRefEditorService.java [hunk 3]
```diff
@@ -456,116 +420,20 @@ private String computeDaiValue(AbstractLNAdapter<?> lnAdapter, TExtRef extRef, S
...
-
-
-    @Override
-    public List<SclReportItem> updateIedNameBasedOnLnode(SCL scl) {
-        Map<TopoKey, TBay> bayByTopoKey = scl.getSubstation().stream()
-                .flatMap(tSubstation -> tSubstation.getVoltageLevel().stream())
-                .flatMap(tVoltageLevel -> tVoltageLevel.getBay().stream())
-                .map(tBay -> PrivateUtils.extractCompasPrivate(tBay, TCompasTopo.class)
-                        .filter(tCompasTopo -> isNotBlank(tCompasTopo.getNode()) && Objects.nonNull(tCompasTopo.getNodeOrder()))
-                        .map(tCompasTopo -> new BayTopoKey(tBay, new TopoKey(tCompasTopo.getNode(), tCompasTopo.getNodeOrder())))
```
[the remark this answers is not in the document]
Comment (samirromdhani) [reply sct-commons/src/main/java/org/lfenergy/compas/sct/commons/ExtRefEditorService.java:510]: done

File: sct-commons/src/main/java/org/lfenergy/compas/sct/commons/ExtRefEditorService.java [hunk 4]
```diff
@@ -38,18 +35,10 @@
 @RequiredArgsConstructor
 public class ExtRefEditorService implements ExtRefEditor {
```
[the remark this answers is not in the document]
Comment (samirromdhani) [reply sct-commons/src/main/java/org/lfenergy/compas/sct/commons/ExtRefEditorService.java:32]: done

File: sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ied/InputsAdapterTest.java (hunk 1 above)
Comment (samirromdhani) [reply sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ied/InputsAdapterTest.java:311]: done

**Pull request merged**